tests/data_context/store/test_validation_config_store.py

```diff
@@ -142,3 +145,100 @@ def test_get_key_cloud(cloud_backed_store: ValidationConfigStore):
     key = cloud_backed_store.get_key(name="my_validation")
     assert key.resource_type == GXCloudRESTResource.VALIDATION_CONFIG
     assert key.resource_name == "my_validation"
+
+
+@pytest.mark.cloud
+@pytest.mark.parametrize(
+    "response_json",
+    [
+        pytest.param(
+            {
+                "data": {
+                    "id": "a4sdfd-64c8-46cb-8f7e-03c12cea1d67",
```

Comment:
this is a lot to look at. Would it make sense to pull out the validation config dict into a constant?

Reply:
Might help to pull the id out into a constant to make the comment below less needed.